cmake/test.cmake

```diff
@@ -204,6 +204,8 @@ finally:
          "${CMAKE_INSTALL_PREFIX}/plugin/adsk/plugin")
     list(APPEND MAYAUSD_VARNAME_PYTHONPATH
          "${CMAKE_INSTALL_PREFIX}/plugin/adsk/scripts")
+    list(APPEND MAYAUSD_VARNAME_MAYA_SCRIPT_PATH
```

Comment:
Fixes the
Error: line 1: Cannot find procedure "mayaUsd_pluginBatchLoad".
error when running tests.

Reply:
great thanks.